inlong-sdk/sort-sdk/src/main/java/org/apache/inlong/sdk/sort/impl/InLongTopicManagerImpl.java

```diff
@@ -100,6 +106,25 @@ public InLongTopicFetcher addFetcher(InLongTopic inLongTopic) {
         }
     }
 
+    /**
+     * create fetcher (pulsar,tube,kafka)
+     *
+     * @param inLongTopic {@link InLongTopic}
+     * @return {@link InLongTopicFetcher}
+     */
+    private InLongTopicFetcher createInLongTopicFetcher(InLongTopic inLongTopic) {
+        if (InlongTopicTypeEnum.PULSAR.getName().equals(inLongTopic.getTopicType())) {
+            logger.info("the topic is pulsar {}", inLongTopic);
+            return new InLongPulsarFetcherImpl(inLongTopic, context);
+        } else if (InlongTopicTypeEnum.TUBE.getName().equals(inLongTopic.getTopicType())) {
```

Comment:
Is it better to use switch/case replacing if/else.

Reply:
Here, the type and type of topic are predictable, and switch case is not required